### ISSUE TITLE
improve commit window speed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [guberm/tortoisegit#1861]
```
Just used tgit on Win XP and was surprised to see how much faster it is on a 5 
years old laptop then on my beefy Win 7 desktop.

What steps will reproduce the problem?
1. commit a file with /closeonend on WinXP vs Win 7
2. measure the time it takes from the windows to disappear on the screen
3.

What is the expected output? What do you see instead?
I would have expected the Win 7 variant to be at least as fast as the one on 
WinXP. It's quite the opposite - I suspect that's because on Win7 it takes some 
time to draw and fill the completed bar which adds a 1-2 seconds delay probably 
due to the Aero skin.
On XP, things happen instantly - the commit window simply blinks and you're off.

Basically tgit on XP is faster then on Win7. It would be nice to improve this 
or have an option that eliminates the lag on all windows system regardless of 
their version or skin.

What version of TortoiseGit and msysgit are you using? On
what operating system?
Tgit 1.7.15, 1.8.4 and 1.8.5 on Win XP and Win 7 X64 SP1

Please provide any additional information below.


```

Original issue reported on code.google.com by `costin.l...@gmail.com` on 20 Aug 2013 at 8:42
